ControlCenterUi/src/types.ts

```diff
@@ -11,3 +11,7 @@ export interface HsvRequest extends LightingRequest {
   s?: number;
   v?: number;
 }
+
+export interface ColorTempRequest extends LightingRequest {
```

Comment:
Nice

Reply:
Nice job extending the other interface that is *

Please rename to BulbTemperatureRequest for clarity